crates/dbsp/src/trace/ord/file/val_batch.rs

```diff
@@ -380,10 +378,6 @@ where
     weight_factory: &'static dyn Factory<R>,
     key_cursor: RawKeyCursor<'s, K, V, T, R>,
     val_cursor: RawValCursor<'s, K, V, T, R>,
-    key: Box<K>,
```

Comment:
so these have now moved inside the cursors?

Reply:
yes